### PR TITLE
Create feature2.txt

### PR DESCRIPTION
**Pull request by suresh9403**

### CODE REVIEW

File: feature2.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+feature2.txt
```
Comment (gajun755) [inline feature2.txt:1]: check spelling

**Review by gajun755 — CHANGES_REQUESTED** [review]
dsd

File: feature2.txt (hunk 1 above)
Comment (suresh9403) [reply feature2.txt:1]: resolved

**Comment by gajun755** [comment]
all done

**Review by gajun755 — APPROVED** [review]
approve

**Comment by gajun755** [comment]
fdfd

**Comment by gajun755** [comment]
please approve

**Pull request merged**